VERSION

```diff
@@ -1 +1 @@
-v0.2.0
+v0.2.1
```

Comment:
since you added support for new features (and changed the interface), please consider using v0.3.0 instead. It's ok if we end up incrementing the minor version number for every change...